homeassistant/components/haveibeenpwned/sensor.py

```diff
@@ -160,7 +160,7 @@ def update(self, **kwargs):
         """Get the latest data for current email from REST service."""
         try:
             url = f"{URL}{self._email}?truncateResponse=false"
-            header = {USER_AGENT: HA_USER_AGENT, "hibp-api-key": self._api_key}
+            header = {str(USER_AGENT): HA_USER_AGENT, "hibp-api-key": self._api_key}
```

Comment:
Since we're using `requests` and not `aiohttp` I think we should not use the name from `aiohttp.hdrs` here at all.

```suggestion
            header = {"User-Agent": HA_USER_AGENT, "hibp-api-key": self._api_key}
```

Please remove the aiohttp import too.